Hymns/Infrastructure/View+Extensions.swift

```diff
@@ -22,6 +22,10 @@ extension View {
     func resignKeyboardOnDragGesture() -> some View {
         return modifier(ResignKeyboardOnDragGesture())
     }
+
+    func maxSize(alignment: Alignment = .center) -> some View {
```

Comment:
If you have suggestions for a better name for this method, pls suggest!

Reply:
optimalSize? Lol idk